js/apps/admin-ui/src/components/keycloak-card/ClickableCard.tsx

```diff
@@ -1,13 +1,15 @@
 import { KeyboardEvent } from "react";
-import { Card, CardProps } from "@patternfly/react-core";
+import { Card, CardHeader, CardProps } from "@patternfly/react-core";
 
 type ClickableCardProps = Omit<CardProps, "onClick"> & {
+  id: string;
   onClick: () => void;
 };
 
 export const ClickableCard = ({
-  children,
+  id,
```

Comment:
Instead of passing the `id` as a prop this could be handled internally by using the [`useId()`](https://react.dev/reference/react/useId) hook.

```ts
const id = useId();
```